src/ef_aws_resolver.py

```
@@ -458,6 +458,20 @@ def cloudfront_origin_access_identity_oai_canonical_user_id(self, lookup, defaul
       else:
         return default
 
+  def kms_decrypt_value(self, lookup, default=None):
+    """
+    Args:
+      lookup: the encrypted value to be decrypted by KMS; base64 encoded
+      default: the optional value to return if lookup failed; returns None if not set
+    Returns:
+      The decrypted key value
+    """
+    try:
+      decrypted_value = EFAwsResolver.__CLIENTS["kms"].decrypt(
+        CiphertextBlob=lookup.decode('base64'))['Plaintext']
+    except ClientError:
+      return default
```

Yeah, I totally agree. This was just me trying to follow what seemed to be the established pattern. Do we want to throw an exception if we can't decrypt the value?